config/locales/diaspora/en.yml

```diff
@@ -1106,6 +1106,8 @@ en:
   shared:
     aspect_dropdown:
       add_to_aspect: "Add contact"
+      mobile_row_checked: "✓ %{name} (remove)"
+      mobile_row_unchecked: "– %{name} (add)"
```

Comment:
In that case you could add the additional char in `app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml` and `app/assets/javascripts/mobile/profile_aspects.js`. I just think those chars shouldn't be in the translated string.